addon/mixins/table-header.js

```diff
@@ -66,13 +67,22 @@ export default Ember.Mixin.create({
   resizeOnDrag: false,
 
   /**
+   * CSS classes to be applied to an `<i class="lt-sort-icon></i>` tag that is
+   * inserted into the column's `<th>` element.
+   *
+   * For instance, if you have installed `ember-font-awesome`, you can set
+   * `iconAscending` to `'fa fa-sort-asc'`, which would yield this markup:
+   * `<i class="lt-sort-icon fa fa-sort-asc"></i>`
+   *
```

Comment:
in lieu of having documentation that explains what icon fonts are & how they work (I couldn't find anything on the web), this well explained documentation with an example will do :+1: